include/clover_field.h

```diff
@@ -37,8 +37,9 @@ namespace quda {
     void *cloverInv;
```

Comment:
@maddyscientist Wanted to ask a strategic question again, sorry for the bother.... I've generalised the clover inverse to support the epsilon^2 piece for non-deg twisted clover by carrying a `QudaTwistFlavorType` in the param struct (instead of the `bool twisted`). This of course affects various interface points and I'm not sure if this is acceptable. In addition I could modify the template signature of the corresponding kernel to avoid the unneccesarry `-arg.epsilon2` below:

```
        if (Arg::twist) { // Compute (T^2 + mu2 - epsilon2) first, then invert
          A = A.square();
          // not sure what's better, a scalar -0.0 for the generate case or another
          // if condition and a change to the template arguments of CloverInvertArg
          A += (arg.mu2 - arg.epsilon2);
        }
```

but I figured that a scalar subtraction is not a big deal here.

I'm unsure about the collateral that I'm causing with this change and would really appreciate a heads up if I should adapt the strategy.

Reply:
I also wanted to avoid having to touch the tuning string for the clover inverse kernel.